pkgs/development/libraries/libdatachannel/default.nix

```
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+, libnice
+, nlohmann_json
+, openssl
+, srtp
+, usrsctp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libdatachannel";
+  version = "0.18.4";
```

```suggestion
  version = "0.18.5";
```
New version has been released in the meantime